src/deploy/k8s/deploy-scdf.sh

```diff
@@ -137,12 +137,11 @@ if [ "$K8S_DRIVER" != "tmc" ] && [ "$K8S_DRIVER" != "gke" ]; then
 
     if [ "$USE_PRO" = "true" ]; then
         sh "$SCDIR/load-image.sh" "dev.registry.tanzu.vmware.com/p-scdf-for-kubernetes/scdf-pro-server:$DATAFLOW_PRO_VERSION" true
-#        if [[ "$DATAFLOW_PRO_VERSION" == *"1.6"* ]]; then
-#            sh "$SCDIR/load-image.sh" "dev.registry.tanzu.vmware.com/p-scdf-for-kubernetes/scdf-pro-skipper:$DATAFLOW_PRO_VERSION" true
-#
-#        else
+        if [[ "$DATAFLOW_PRO_VERSION" == *"1.6"* ]]; then
+            sh "$SCDIR/load-image.sh" "dev.registry.tanzu.vmware.com/p-scdf-for-kubernetes/scdf-pro-skipper:$DATAFLOW_PRO_VERSION" true
```

Comment:
So this change is so that we use the PRO instead of the Skipper OSS version if the dataflow is 1.6?  Is there a way to check for 1.6 or greater?   So we don't have to update it in the future.  Thoughts?